api-client/src/errors/NotFoundError.ts

```diff
@@ -30,11 +30,13 @@ class NotFoundError extends Error {
       Error.captureStackTrace(this, NotFoundError)
     }
 
+    // default values are provided for cases where parameters are unknown
+    // e.g. when throwing NotFoundError from an endpoint
     this.name = 'NotFoundError'
-    this._type = params.type
     this._id = params.id
-    this._city = params.city
-    this._language = params.language
+    this._type = params.type
+    this._city = params.city ?? 'unknown city'
+    this._language = params.language ?? 'en'
```

Comment:
So the best solution would probably be to just remove `city` and `language` from the not found error completely and pass it separately as props if necessary. However, this would make changes in a lot of place necessary and is not worth the effort right now.
Therefore I'd prefer just making the properties optional and not using any fallback. Then you'd just have to handle the case that there is no city or language separately in `FailureSwitcher` by adding that as a condition to the if clause but everything else should continue to work.